source/mongorocks.rst

```diff
@@ -21,10 +27,11 @@ Using MongoRocks
 ================
 
 As of version 3.2, |PSMDB| runs with `WiredTiger`_ by default.
-You can select a storage engine
-using the ``--storageEngine`` command-line option when you start ``mongod``.
-Alternatively, you can set the ``storage.engine`` variable
-in the configuration file (by default, :file:`/etc/mongod.conf`):
+You can select a storage engine using the ``--storageEngine``
```

Comment:
This needs a lead-in. For example "If you still would like to use the deprecated MongoRocks storage engine, please use the ``--storageEngine rocksdb`` command-line option, accompanied by ...."